_includes/scenarios/integrate/requests-over-templates.md

```diff
@@ -62,35 +65,37 @@ You will receive a response with the complete RelationshipTemplate.
+You can observe this by [long polling the incoming Requests]({% link _docs_use-cases/use-case-consumption-query-incoming-requests.md %}) and optionally use the query params `source.reference=<id-of-the-relationshiptemplate>` and `status=ManualDecisionRequired` to filter for Requests that belong to the RelationshipTemplate you are currently working on.
 
-For more information about the events you can head over to the [Connector Modules site]({% link _docs_operate/modules.md %}) and read about the [AMQP Publisher module]({% link _docs_operate/modules.md %}#amqppublisher) and the [Webhooks module]({% link _docs_operate/modules.md %}#webhooks) that are propagating events.
+In a productive environment, however, we recommend using the [Sync module]({% link _docs_operate/modules.md %}#sync) and waiting for a `consumption.incomingRequestReceived` [Connector Event]({% link _docs_integrate/connector-events.md %}).
+To learn more about events, how to use them in the context of enmeshed and which [modules]({% link _docs_operate/modules.md %}) to help you automating your business process are supported by enmeshed, check out our [Event introduction]({% link _docs_integrate/event-introduction.md %}).
 
 {% include copy-notice description="After you received the Request, save its `id` for the next step." %}
 
 ## Answer the Request
 
-The rejection is explained before the acceptance because you can re-do it as often if you want. If you accept the Request a RelationshipRequest will be sent and no new Request will be created until the RelationshipRequest is answered. If the RelationshipRequest is accepted the enmeshed Runtime will recognize the existing Relationship and will also not create a new Request.
-
-So if you want to test the full flow, you should first reject the Request. After that you can create a new one, which you can accept.
-
-If there is no open RelationshipRequest or existing Relationship, you can trigger the creation of a new Request by [loading the Template again](#load-the-template-and-get-the-request) with the same truncated reference.
+If you want to test the full flow, we recommend to reject the Request first.
+Afterwards, you can create a new one by [loading the RelationshipTemplate](#load-the-relationshiptemplate-and-get-the-request) again and accept it.
 
 ### Reject
 
-If you want to reject the Request you can do so by calling the `PUT /api/v2/Requests/Incoming/{id}/Reject` route. You can use the `id` you saved in the previous step. In the payload you have to reject all RequestItems. In case of the example Request the payload is the following:
+Let's consider the case the Recipient wants to [reject the Request]({% link _docs_use-cases/use-case-consumption-reject-incoming-request.md %}).
+For this, use the `id` of the Request you saved in the previous step.
+In the payload you have to reject all RequestItems.
+In case of the example Request, the payload is the following:
```

Comment:
Sometimes we use a colon before JSON payloads, sometimes not. For reasons of consistency, we should decide for one option, at least within a single scenario page.